docs/internationalization/custom_integration.md

```diff
@@ -4,9 +4,9 @@ title: "Custom Component Localization"
 
 ## Translation Strings
 
-Unlike localized strings merged in the `home-assistant` repository, custom components cannot take advantage of Lokalise for user-submitted translations. However, custom component authors can still include translations with their components. These will be read from the `translations` directory, adjacent to the component source. They are named `<platform name>.<language_code>.json`, e.g., for the German translation of a sensor `sensor.de.json`, unless the translation is for the custom component only, in which case the file is named `<language_code>.json` in the `translations` directory.
+Unlike localized strings merged in the `home-assistant` repository, custom components cannot take advantage of Lokalise for user-submitted translations. However, custom component authors can still include translations with their components. These will be read from the `translations` directory, adjacent to the component source. They are named `<language_code>.json` in the `translations` directory, e.g., for the German translation `de.json`.
 
-These files follow the same formatting as [backend translation string files](internationalization/core.md), but a copy will exist for each translated language.
+This file will contain the different strings that will be translatable for different things that the custom components offers that need to be translated. These files follow the same formatting as [backend translation string files](internationalization/core.md), but a copy will exist for each translated language.
 
 The language codes follow the [BCP47](https://tools.ietf.org/html/bcp47) format. The [frontend translation files](https://github.com/home-assistant/frontend/tree/master/src/translations) can also be referred to if you are unsure of the correct language code to use.
```

Comment:
There are no more files for each language on the frontend, do you think it's better to put the link to the [`translationMetadata.json`](https://github.com/home-assistant/frontend/blob/master/src/translations/translationMetadata.json) file or leave it the way it is?

Reply:
I would remove the last sentence in that paragraph.

```suggestion
The language codes follow the [BCP47](https://tools.ietf.org/html/bcp47) format.
```